packages/fleather/lib/src/widgets/text_selection.dart

```diff
@@ -247,28 +211,36 @@ class EditorTextSelectionOverlay {
   }
 
   /// Whether the handles are currently visible.
-  bool get handlesAreVisible => _handles != null && handlesVisible;
+  bool get handlesAreVisible => _handles != null;
```

Comment:
Will be *unused* with comment in 'editor_selection_delegate_mixin.dart` L84

Reply:
Since `EditorTextSelectionOverlay` is exposed from `EditorState` and `_handles` are private, I think it's better to keep it. WDYT? @amantoux 
```dart
final key = GlobalKey<EditorState>();
print(key.currentState.selectionOverlay.handlesAreVisible);
```

Reply:
My point is, as long as we do not need it we should remove it.

Reply:
Is it ok now? But beside internal usages, since it's public, it might be used outside of Fleather. @amantoux

Reply:
Why? 😁 @amantoux